helpers/calendar/events.py

```diff
@@ -56,8 +56,8 @@ def get_room_schedules(self, calendar_id, days):
                         all_attendees.append(attendee.get('email'))
         return [all_attendees, output]
 
-    def get_all_room_schedules(self, query, start_date, end_date):
-        rooms = query.filter_by(state="active")
+    def get_all_room_schedules(self, query, start_date, end_date, location_id):
```

Comment:
Great Job @daniellamarr. Please, could you provide documentation for the function you just created?. I feel this will help other people understand the code better

Reply:
@Makwe-O I didn't create a function, I just added the `location_id` parameter to the pre-existing function so it can be used to filter room data based on the location of the user.

Reply:
But I guess you're right. I'd go ahead and add a documentation

Reply:
I have made the changes